src/main/resources/assets/sass/_breadcrumbs.scss

```diff
@@ -0,0 +1,19 @@
+// Breadcrumbs
+// ==========================================================================
+
+.breadcrumbs {
+  @include breadcrumbs;
+
+  ol {
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    background-image: image-url("images/seperator.png");
+
+    @include device-pixel-ratio() {
```

Comment:
I haven't seen this mixin before but I assume it means "retina" when used without arguments.